server.py

```diff
@@ -31,18 +35,20 @@ def server():
         server_socket.close()
 
 
-def response_ok(msg):
-    result = "HTTP/1.1 200 OK\r\n"
-    con_type = "Content-Type: text/plain\r\n"
-    body = "Content length: {}".format(len(msg))
-    return "{}{}{}".format(result, con_type, body)
+def response_ok(msg, resolved):
+    result = "\nHTTP/1.1 200 OK\r\n"
+    timestamp = 'Date: {}\r\n'.format(email.utils.formatdate(usegmt=True))
+    con_type = "Content-Type: {}".format(msg)
+    body = "{}".format(resolved)
+    return "{}{}{}{}".format(result, timestamp, con_type, body)
```

Comment:
You're missing a blank line between the HTTP headers and the body of the HTTP response.  Without that blank line your browser will not properly interpret this response.